src/main/resources/application.yaml

```diff
@@ -26,8 +26,8 @@ spring:
     password: ${AM_DB_PASSWORD:ampass}
     properties:
       charSet: UTF-8
-  flyway:
-    enabled: ${ENABLE_DB_MIGRATE_IN_SERVICE:true}
+  flyway.locations: classpath:db/migration,classpath:db/migrationAks
+  flyway.enabled: ${ENABLE_DB_MIGRATE_IN_SERVICE:true}
```

Comment:
Please change this to YAML syntax as per the rest of the file (ie. no full stops)